src/Catel.Core/Catel.Core.Shared/Data/PropertyBag.cs

```diff
@@ -135,7 +135,13 @@ public TValue GetPropertyValue<TValue>(string propertyName, TValue defaultValue)
 
             lock (_lockObject)
             {
-                return _properties.ContainsKey(propertyName) ? (TValue)_properties[propertyName] : defaultValue;
+                object propertyValue;
+                if (this._properties.TryGetValue(propertyName, out propertyValue))
```

Comment:
Please remove the "this." to keep a consistent coding style throughout the code base